pkg/integrations/windows_exporter/windows_exporter.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
+
+package windows_exporter //nolint:golint
+
+import (
+	"context"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/gorilla/mux"
+	"github.com/grafana/agent/pkg/integrations/config"
+)
+
+// Integration is the windows_exporter integration. On non-Windows platforms,
+// this integration does nothing and will print a warning if enabled.
+type Integration struct {
+	c *Config
```

Comment:
You don't need to store the config here, feel to just ignore both arguments to New.